### ISSUE TITLE
grammy command can create an unbounded number of parallel network requests

### ISSUE THREAD

**Issue by starpit** [IBM/kui#7151]
If i point grammy to a set of remote VFS filepaths, it will fetch them all in parallel. We should probably bound the concurrency.

Also, I think we can increase performance a bit by using fslice rather than fstat. Since all we need is the data.

**Comment by starpit** [comment]
It might also at some point be nice to have grammy product a live table, so that we don't need to wait for grammy to fetch all of the data before we can see a histogram for some of the data.
